lib/seahorn/HornifyModule.cc

```diff
@@ -372,9 +440,35 @@ bool HornifyModule::runOnFunction(Function &F) {
   /// -- allocate LiveSymbols
   auto r = m_ls.insert(std::make_pair(&F, LiveSymbols(F, m_efac, *m_sem)));
   assert(r.second);
+
+  // HACK because reset counters because "run()" calls VisitCallSite
+  // TODO: store part of what is computed by LiveSymbols?
+  unsigned tmp_m_n_params = m_n_params;
```

Comment:
create a class for all the counters, say called `stats` and manipulate that class instead of the individual counters.

Reply:
done